Utilities/File.cpp

```diff
@@ -231,44 +234,22 @@ bool fs::create_path(const std::string& path)
 {
 	g_tls_error = fse::ok;
 
-	size_t start = 0;
-
-	while (true)
+	std::string parent;
 	{
-		// maybe it could be more optimal if goes from the end recursively
-		size_t pos = path.find_first_of("/\\", start);
-
-		if (pos == std::string::npos)
-		{
-			pos = path.length();
-		}
-
-		std::string dir = path.substr(0, pos);
-
-		start = ++pos;
-
-		if (dir.size() == 0)
-		{
-			continue;
-		}
-
-		if (!is_dir(dir))
-		{
-			// if doesn't exist or not a dir
-			if (!create_dir(dir))
-			{
-				// if creating failed
-				return false;
-			}
-		}
-
-		if (pos >= path.length())
-		{
-			break;
-		}
+#ifdef _WIN32
+		auto copy = to_wchar(path);
+		PathRemoveFileSpecW(copy.get());
```

Comment:
It seems it doesn't work with "/", only with "\". I may be wrong, but I think it can be solved better with `std::string::find_last_of`